core-bundle/src/Resources/contao/library/Contao/Template.php

```diff
@@ -395,6 +399,32 @@ public function trans($strId, array $arrParams=array(), $strDomain='contao_defau
 		return System::getContainer()->get('translator')->trans($strId, $arrParams, $strDomain);
 	}
 
+	/**
+	 * Adds JSON-LD data to the current page.
+	 */
+	public function jsonLd(array $jsonLd, string $schema = JsonLdManager::SCHEMA_ORG): void
```

Comment:
What about naming this `addJsonLd()` instead? To me that would sound less as it would return something.

Reply:
Was this closed intentionally? :slightly_smiling_face:

Reply:
Sorry, missed that!